Combine pod creation with IDP registration (#749)
solid/community-server: config/http/README.md

## Handler
Sets up all the handlers a request will potentially pass through.
* *default*: The full setup, that is middleware + static files + IDP + LDP.

> * *simple*: A simpler setup that only supports the base solid features. Has no IDP.

Reviewer: ```suggestion
* *simple*: A simpler setup in which the IDP is disabled.
```
